%&<latex>
\documentclass[letterpaper,12pt]{article}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% preamble %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\input{../utils/preamble.tex}
\input{../utils/macros.tex}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}
\doublespacing
\raggedright
\setlength{\parindent}{0.5in}
\begin{linenumbers}

\begin{titlepage}
    \begin{flushleft}
        \sffamily

        \MakeUppercase{\large\bfseries Title}

        \vspace{12pt}
        \textbf{Running head:} \MakeUppercase{Running head}

        \vspace{12pt}
        Jamie R.\ Oaks$^{1,2}$ and Mark T.\ Holder$^{1}$

        \bigskip
        $^1$\emph{Department of Ecology and Evolutionary Biology,
            % Biodiversity Institute,
            University of Kansas,
            Lawrence, Kansas 66045}\\[.1in]
        $^2$\emph{Corresponding author} (\href{mailto:joaks1@gmail.com}{\tt
        joaks1@gmail.com})\\

    \end{flushleft}
\end{titlepage}

{\sffamily
    \noindent\textbf{ABSTRACT} \\
    \noindent Abstract here \ldots

    \vspace{12pt}
    \noindent\textbf{KEY WORDS: } 
}

\newpage
\noindent \ldots

\section*{Methods}

\subsection*{The model}
We follow much of the notation of \citet{Oaks2012}, but modify it to
accommodate the new model \ldots

We assume an investigator is interested in inferring the distribution
of divergence times among \npairs{} pairs of populations.
For each pair $i$, \popSampleSize{i}{} genome copies have been sampled, with
\popSampleSize{1}{i} copies sampled from population 1, and \popSampleSize{2}{i}
sampled from population 2.
From these genomes, let \nloci{i} be the number of DNA sequence loci collected
for population pair $i$, and \nlociTotal be the total number of unique loci
sampled across the \npairs{} pairs of populations.
We use \alignment{i}{j} to represent the multiple sequence alignment of
locus $j$ for population pair $i$.
$\alignmentVector = (\alignment{1}{1}, \ldots,
    \alignment{\npairs{}}{\nloci{\npairs{}}})$
is the full dataset,
i.e., a vector of sequence alignments for all pairs and loci.
We let \geneTree{i}{j} represent the gene tree upon which \alignment{i}{j}
evolved according to fixed \hky substitution model parameters \hkyModel{i}{j}.
The investigtor must specify the parameters of all
$\hkyModelVector = (\hkyModel{1}{1}, \ldots,
\hkyModel{\npairs{}}{\nloci{\npairs{}}})$
substitution models by which the alignments evolved along the
$\geneTreeVector = (\geneTree{1}{1}, \ldots,
\geneTree{\npairs{}}{\nloci{\npairs{}}})$
gene trees.
Furthermore, the investigator must specify a vector of fixed contants
$\ploidyScalarVector = (\ploidyScalar{1}{1}, \ldots,
\ploidyScalar{\npairs{}}{\nloci{\npairs{}}})$
that scale the population size parameters for known differences in ploidy among
loci and/or differences in generation times among population pairs.
Lastly, the investigator must also specify a vector of fixed constants
$\mutationRateScalarConstantVector = (\mutationRateScalarConstant{1}{1},
\ldots, \mutationRateScalarConstant{\npairs{}}{\nloci{\npairs{}}})$
that scale the population size parameters for known differences in
mutation rates among loci and/or among taxa.

\begin{linenomath}
With \alignmentVector, \hkyModelVector, \ploidyScalarVector, and
\mutationRateScalarConstantVector in hand, the joint posterior distribution
of our model is given by
\begin{equation}
    p(\geneTreeVector, \divTimeMapVector, \demographicParamVector, 
    \locusMutationRateScalarVector, \locusRateHetShapeParameter \given
    \alignmentVector, \hkyModelVector, \ploidyScalarVector,
    \mutationRateScalarConstantVector) =
    \frac{p(\alignmentVector \given \geneTreeVector, \hkyModelVector)
        p(\geneTreeVector \given \divTimeMapVector, \demographicParamVector,
        \locusMutationRateScalarVector, \ploidyScalarVector,
        \mutationRateScalarConstantVector)
        p(\locusMutationRateScalarVector \given \locusRateHetShapeParameter)
        p(\locusRateHetShapeParameter)
        p(\divTimeMapVector)
        p(\demographicParamVector)}{
        p(\alignmentVector)},
    \label{eq:fullModelCompact}
\end{equation}
where
$\divTimeMapVector = (\divTimeMap{1}, \ldots, \divTimeMap{\npairs{}})$
is a vector of population divergence times for each of \npairs{} pairs,
$\demographicParamVector = (\demographicParams{1}, \ldots,
\demographicParams{\npairs{}})$
is a vector of the demographic parameters for each of the \npairs{} population
pairs,
$\locusMutationRateScalarVector = (\locusMutationRateScalar{1}, \ldots
\locusMutationRateScalar{\nlociTotal})$
is a vector of locus-specific mutation-rate scaling parameters for each of the
\nlociTotal loci,
and \locusRateHetShapeParameter is the shape parameter of a gamma-distributed
prior on \locusMutationRateScalar{}.
\end{linenomath}

\subsubsection*{Likelihood terms of Equation \ref{eq:fullModelCompact}}
\begin{linenomath}
The likelihood terms of Equation \ref{eq:fullModelCompact} can
be expanded out as a product over population pairs and loci
\begin{equation}
    p(\alignmentVector \given \geneTreeVector, \hkyModelVector)
    p(\geneTreeVector \given \divTimeMapVector, \demographicParamVector,
    \locusMutationRateScalarVector, \ploidyScalarVector,
    \mutationRateScalarConstantVector) = 
    \prod_{i=1}^{\npairs{}}
    \prod_{j=1}^{\nloci{i}}
    p(\alignment{i}{j} \given \geneTree{i}{j}, \hkyModel{i}{j})
    p(\geneTree{i}{j} \given \divTimeMap{i}, \demographicParams{i},
    \locusMutationRateScalar{j}, \ploidyScalar{i}{j},
    \mutationRateScalarConstant{i}{j}).
    \label{eq:modelLikelihoodExpanded}
\end{equation}
The first term,
$p(\alignment{i}{j} \given \geneTree{i}{j}, \hkyModel{i}{j})$,
is the probability of the sequence alignment of locus $j$ for population pair
$i$ given the gene tree and \hky substitution model paramters \citep[i.e., the
``Felsenstein likelhood'';][]{Felsenstein1981}.
The model allows for an intra-locus recombination rate \recombinationRate,
which, for simplicity, is assumed to be zero in Equation
\ref{eq:fullModelCompact}.
If \recombinationRate is non-zero, this term requires an additional product
over the columns (sites) of each sequence alignment to allow sites to have
different genealogies.
The second term,
p(\geneTree{i}{j} \given \divTimeMap{i}, \demographicParams{i},
\locusMutationRateScalar{j}, \ploidyScalar{i}{j},
\mutationRateScalarConstant{i}{j}),
is the probability of the gene tree under a multi-population coalescent model
(i.e., species tree) where the ancestral population of pair $i$ diverges
and gives rise to the two sampled descendant populations.
Each \demographicParams contains the following demographic parameters:
The mutation-scaled sizes of the ancestral, \ancestralTheta{}, and descendant
populations, \descendantTheta{1}{} and \descendantTheta{2}{};
the proportion of the first, \bottleScalar{1}{}, and second population,
\bottleScalar{2}{}, that persists during bottlenecks that begin immmediately
after divergence in forward-time;
the proportion of time between present and divergence when the bottlenecks
end for both populations, \bottleTime{};
and the symmetric migration rate between the descendant populations,
\migrationRate{}.
Thus, the probability of the $\popSampleSize{i}{}-1$ coalescence times (node
heights) of gene tree \geneTree{i}{j} is given by a multi-population
Kingman-coalescent model \citep{Kingman1982} where the ancestral population of
size
$\ancestralTheta{i}\ploidyScalar{i}{j}\mutationRateScalarConstant{i}{j}\locusMutationRateScalar{j}$
diverges at time \divTimeMap{i} into two descendant populations of constant
size
$\descendantTheta{1}{i}\ploidyScalar{i}{j}\mutationRateScalarConstant{i}{j}\locusMutationRateScalar{j}\bottleScalar{1}{i}$
and
$\descendantTheta{2}{i}\ploidyScalar{i}{j}\mutationRateScalarConstant{i}{j}\locusMutationRateScalar{j}\bottleScalar{2}{i}$,
which, after time $\divTimeMap{i}\bottleTime{i}$, grow exponentially to their
present size 
$\descendantTheta{1}{i}\ploidyScalar{i}{j}\mutationRateScalarConstant{i}{j}\locusMutationRateScalar{j}$
and
$\descendantTheta{2}{i}\ploidyScalar{i}{j}\mutationRateScalarConstant{i}{j}\locusMutationRateScalar{j}$,
respectively.
Following divergence, the descendant populations of pair $i$ exchange migrants at a symmetric rate of
\migrationRate{i}.
\end{linenomath}

\subsubsection*{Prior terms of Equation \ref{eq:fullModelCompact}}
\begin{linenomath}
The term $p(\locusRateHetShapeParameter)$ is the prior density
function for the shape-parameter for the gamma-distributed prior on
rate heterogeneity among loci.
This prior is $\locusRateHetShapeParameter \sim U(1, 20)$.
The prior probability of the vector of locus-specific mutation-rate scaling parameters
given \locusRateHetShapeParameter then expands out as a product over
the loci
\begin{equation}
    p(\locusMutationRateScalarVector \given \locusRateHetShapeParameter) =
    \prod_{j=1}^{\nlociTotal}
    p(\locusMutationRateScalar{j} \given \locusRateHetShapeParameter),
    \label{eq:locusRateHetPrior}
\end{equation}
where each \locusMutationRateScalar{} is independently and identicially
distributed (\iid) as
$\locusMutationRateScalar{} \sim Gamma(\locusRateHetShapeParameter,
1/\locusRateHetShapeParameter)$.
If the recombination rate \recombinationRate is allowed to be
non-zero, the prior term $p(\recombinationRate)$ would be added
to Equation \ref{eq:fullModelCompact}, and the prior would be
$\recombinationRate \sim Gamma(\gammaShape{\recombinationRate},
\gammaScale{\recombinationRate})$.
\end{linenomath}

\begin{linenomath}
The prior term for the demographic parameters, $p(\demographicParamVector)$,
expands out into its components and as a product over the \npairs{}
pairs of populations
\begin{equation}
    p(\demographicParamVector) =
    \prod_{i=1}^{\npairs{}}
    p(\ancestralTheta{i})
    p(\descendantTheta{1}{i})
    p(\descendantTheta{2}{i})
    p(\bottleScalar{1}{i})
    p(\bottleScalar{2}{i})
    p(\bottleTime{i})
    p(\migrationRate{i}).
    \label{eq:demographicPrior}
\end{equation}
The priors for the demographic parameters are:
$\ancestralTheta{} \sim Gamma(\gammaShape{\ancestralTheta{}},
\gammaScale{\ancestralTheta{}})$,
$\descendantTheta{1}{} \sim Gamma(\gammaShape{\descendantTheta{}{}},
\gammaScale{\descendantTheta{}{}})$,
$\descendantTheta{2}{} \sim Gamma(\gammaShape{\descendantTheta{}{}},
\gammaScale{\descendantTheta{}{}})$,
$\bottleScalar{1}{} \sim Beta(\betaA{\bottleScalar{}{}},
\betaB{\bottleScalar{}{}})$,
$\bottleScalar{2}{} \sim Beta(\betaA{\bottleScalar{}{}},
\betaB{\bottleScalar{}{}})$,
$\bottleTime{} \sim U(0, 1)$,
and
$\migrationRate{} \sim Gamma(\gammaShape{\migrationRate{}},
\gammaScale{\migrationRate{}})$.
\end{linenomath}

\subsubsection*{Priors on divergence models}
\begin{linenomath}
The prior term for the vector of divergence times for each of
the \npairs{} pairs of populations can be expanded as
\begin{equation}
    p(\divTimeMapVector) = p(\divTimeIndexVector)p(\divTimeVector \given \divTimeIndexVector),
    \label{eq:divModelPrior}
\end{equation}
where \divTimeVector is an ordered set of unique divergence time parameters
$\{\divTime{1}, \ldots, \divTime{\divTimeNum}\}$ whose length
\divTimeNum can range from 1 to \npairs{},
and \divTimeIndexVector is vector of indices of divergence time parameters 
$(\divTimeIndex{1}, \ldots, \divTimeIndex{\npairs{}})$
that map the divergence time in \divTimeVector to each of the \npairs{}
pairs of populations.
Thus, \divTimeMapVector is the result of applying the mapping function
\begin{equation}
    f(\divTimeVector, \divTimeIndexVector, i) = \divTime{\divTimeIndex{i}}
    \label{eq:divTimeMapFunction}
\end{equation}
to each population pair $i$, such that
$\divTimeMapVector = (\divTimeMap{1} = f(\divTimeVector, \divTimeIndexVector,
1), \ldots, \divTimeMap{\npairs{}} = f(\divTimeVector, \divTimeIndexVector,
\npairs{}))$.
Each \divTime{} within \divTimeVector is \iid as
$\divTime{} \sim Gamma(\gammaShape{\divTime{}}, \gammaScale{\divTime{}})$.
\end{linenomath}

Biologically speaking, \divTimeVector contains the times of divergence events,
the length of which \divTimeNum is the number of divergence events shared across
the \npairs{} pairs of populations.
For example, if \divTimeVector contains a single divergence time parameter
\divTime{1}, all \npairs{} pairs of populations are constrained to diverge at
this time (i.e., \divTimeIndexVector would contain the index 1 repeated
\npairs{} times, and \divTimeMapVector would contain the value \divTime{1}
repeated \npairs{} times), whereas if it contains \npairs{} divergence time
parameters, the model is fully generalized to allow all of the pairs to diverge
at unique times.

\begin{linenomath}
The total sample space of ordered realizations of \divTimeMapVector is all the
possible partitions of \npairs{} elements into 1 to \npairs{} categories.
The total number of possible partitions is a sum of the Stirling numbers of
the second kind over all possible number of categories \divTimeNum
\begin{equation}
    B_{\npairs{}}=\sum_{\divTimeNum=1}^{\npairs{}} \left[
    \frac{1}{\divTimeNum!} \sum_{j=0}^{\divTimeNum-1} (-1)^{j}
    \binom{\divTimeNum}{j} (\divTimeNum-j)^{\npairs{}} \right],
    \label{eq:bell}
\end{equation}
which is the Bell Number \citep{Bell1934}.
The original \msb model only samples over the unordered realizations of
\divTimeIndexVector, such that the sample space is reduced to all the possible
integer partitions of \npairs{} \citep{Oaks2012,OeisPartitionNumber,
    OeisPartitionTriangle,Malenfant2011}.
This has the advantage of reducing the sample space of the number of
divergence models, but has two disadvantages.
First, additional information about the data is lost.
By throwing away the identity of the \npairs{} pairs of populations, all
information regarding the amount of data (e.g., the number gene copies
collected from each of the populations [\popSampleSize{1}{} and
\popSampleSize{2}{}], the number of loci, and the length of the loci), the
information content in the data (i.e., the amount of sequence variation), as
well as the taxon- and locus-specific parameters (\hkyModel{}{},
\mutationRateScalarConstant{}{}, \ploidyScalar{}{}, and
\locusMutationRateScalar{}) for each pair is lost.
The original descriptions of the \msb model claim that the independence
between the sample sizes \popSampleSize{1}{}, \popSampleSize{2}{} and
the summary statistic $\pi_b$ calculated from the alignment for each
population pair allows exchangeability.
Second, the identity of the pairs of populations is lost.
\end{linenomath}

We implement two prior probability distributions over the joint space of
\divTimeVector, \divTimeIndexVector.
One simply places a uniform prior over all possible integer paritions of
\npairs{}.


\section*{Acknowledgments}
We thank the National Science Foundation for supporting this work (DEB
1011423).
J.\ Oaks was also supported by the University of Kansas (KU) Office of Graduate
Studies, Society of Systematic Biologists, Sigma Xi Scientific Research
Society, KU Department of Ecology and Evolutionary Biology, and the KU
Biodiversity Institute.

\bibliography{../bib/references}

%% LIST OF FIGURES %%%%%%%%%%%%%%%%%%%%%%%%%%
\newpage
\singlespacing

\renewcommand\listfigurename{Figure Captions}
\cftsetindents{fig}{0cm}{2.2cm}
\renewcommand\cftdotsep{\cftnodots}
\setlength\cftbeforefigskip{10pt}
\cftpagenumbersoff{fig}
\listoffigures


\end{linenumbers}

%% TABLES %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newpage
\singlespacing


\clearpage

%% FIGURES %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newpage

%:FIGURE-accuracy omega SS GLM
\begin{figure}[hbtp]
    \captionsetup{listformat=figList}
    \centering
    \includegraphics[width=1.0\textwidth]{../../validation/results/dpp/pymsbayes-results/plots/prior-dpp_prior-dpp_mc_behavior.pdf}
    \caption{
        Model-choice behavior \ldots
    }
    \label{fig_dpp_dpp_val_mc}
\end{figure}

\clearpage


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% SUPPORTING INFO %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\setcounter{figure}{0}
\setcounter{table}{0}
\setcounter{page}{1}

\singlespacing

\renewcommand{\refname}{\noindent\MakeUppercase{\LARGE\sffamily\upshape supporting information}}

% PUT MAIN TEXT CITATION HERE
% \begin{thebibliography}{1}
% \providecommand{\natexlab}[1]{#1}
% \providecommand{\url}[1]{\texttt{#1}}
% \providecommand{\urlprefix}{URL }

% \bibitem

% \end{thebibliography}


%% SUPPL TABLES %%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\setcounter{table}{0}


\end{document}

