\input{../utils/preamble.tex}
\input{../../manuscripts/utils/macros.tex}
\newcommand{\allParameters}[1]{\ensuremath{\theta_{#1}}\xspace}
\bibliography{../../manuscripts/bib/references}

\title[Improving inference of evolutionary history]{Improving inference of evolutionary history}
\subtitle{Developing comparative methods for genomic data}

\author[J.\ Oaks]{
    Jamie R.\ Oaks\inst{1}
}
\institute[University of Washington]{
    \inst{1}%
        Department of Biology, University of Washington
}

% \date{\today}
\date{November 3, 2014}

\begin{document}
% \maketitle
\begin{frame}
    \begin{columns}[c]
        \column{.5\textwidth}
            \maketitle
        \column{.5\textwidth}
            \begin{figure}
                \begin{center}
                \includegraphics[width=\textwidth]{../images/darwin-tol-copyright-boris-kulikov-2007.jpg}
                \caption{\tiny \copyright~2007 Boris Kulikov \href{http://boris-kulikov.blogspot.com/}{boris-kulikov.blogspot.com}}
                \end{center}
            \end{figure}
    \end{columns}
\end{frame}

% I. What I do
% II. What I've been focusing on recently (started in PhD and continuing in postdoc)
%   a. Dynamic planet---need to account for large-scale processes
%       i. E.g., SE Asia---dynamic landscape
{
\setbeamercolor{background canvas}{bg=black}
\setbeamercolor{whitetext}{fg=white}
\begin{frame}
    \frametitle{Processes of diversification}
    \begin{columns}[c]
        \column{.5\textwidth}
        {\usebeamercolor[fg]{whitetext}
        \begin{itemize}
            \item<2-> Large-scale geological and climatic processes are
                important in biodiversification and community assembly
            \item<3-> Accounting for such processes will better our
                understanding of biodiversity
            \item<4-> We need methods for inferring evolutionary patterns
                predicted by historical events from contemporary populations
        \end{itemize}
        }
        \column{.5\textwidth}
        \begin{figure}
            \begin{center}
            \includegraphics[width=\textwidth]{../images/earth-image.jpg}
            \end{center}
        \end{figure}
    \end{columns}
\end{frame}
}

{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{../images/maps/se-asia-present.png}}
\begin{frame}
    \frametitle{Climate-driven diversification}    
\end{frame}
}

{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{../images/maps/se-asia-120.png}}
\begin{frame}
    \frametitle{Climate-driven diversification}    
    \begin{columns}
        \column{0.6\textwidth}
        \ \\

        \column{0.45\textwidth}

        \vspace{-2.4cm}

        \begin{uncoverenv}<2->
        \colorbox{white}{
            \begin{minipage}[t][3.6cm][t]{1.1\textwidth}
                % \begin{onlyenv}<2-3>
                % \begin{itemize}
                %     \item<2-> Repeated coalescence and fragmentation of island complexes
                %     \item<3-> Prominent paradigm for explaining biodiversity
                % \end{itemize}
                % \end{onlyenv}

                \begin{onlyenv}<2>
                \begin{itemize}
                    \item \textbf{Did repeated fragmentation of islands
                        during inter-glacial rises in sea level promote
                        diversification?}
                \end{itemize}
                \end{onlyenv}

                \begin{onlyenv}<3->
                \begin{itemize}
                    \item<3-> Climate-driven model of diversification
                    \item<4-> Testable prediction:
                    \begin{itemize}
                        \item Temporally clustered divergences among taxa
                            co-distributed across fragmented islands
                    \end{itemize}
                \end{itemize}
                \end{onlyenv}
            \end{minipage}
        }
    \end{uncoverenv}
    \end{columns}
\end{frame}
}

% \begin{frame}
%     \frametitle{Philippine Archipelago}
%     \begin{center}
%         \includegraphics<1>[width=.5\textwidth]{../images/maps/Philippines-present.png}
%         \includegraphics<2>[width=.5\textwidth]{../images/maps/Philippines.png}
%     \end{center}
% \end{frame}

% \begin{frame}
%     \frametitle{Climate-driven diversification model}
%     \begin{columns}[c]
%         \column{.5\textwidth}
%             \begin{myitemize}
%                 \item<1-> Repeated coalescence and fragmentation of island complexes
%                 \item<2-> Prominent paradigm for explaining Philippine biodiversity
%                 \item<3-> Proposed as model of diversification
%             \end{myitemize}
%         \column{.5\textwidth}
%             \includegraphics[width=\textwidth]{../images/maps/Philippines.png}
%     \end{columns}
% \end{frame}

% \begin{frame}
%     \frametitle{Testing climate-driven diversification}
%     \begin{columns}[c]
%         \column{.5\textwidth}
%     \textbf{Did repeated fragmentation of islands during inter-glacial rises in sea level
%     promote diversification?}\\
%     \bigskip
%     \uncover<2->{
%     Model has testable prediction:\\
%     \begin{itemize}
%         \item Temporally clustered divergences among taxa
%             co-distributed across fragmented islands
%     \end{itemize}
%     }
%         \column{.5\textwidth}
%             \includegraphics[width=\textwidth]{../images/maps/Philippines.png}
%     \end{columns}
% \end{frame}

%   b. Predictions of climate-driven diversification
\begin{frame}
    \frametitle<1-4>{Climate-driven model: Predictions}
    \frametitle<5->{Divergence model choice}
    \begin{columns}[c]
        \column{.5\textwidth}
        \begin{onlyenv}<1-4>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<5>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeVector = \{\divTime{1}\}
                        \]\vspace{0mm}
                        \[
                            model = 111
                        \]\vspace{0mm}
                        \[
                            \divTimeMapVector = (\divTimeMap{1},\divTimeMap{2},\divTimeMap{3})
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<6>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeVector = \{130\}
                        \]\vspace{0mm}
                        \[
                            model = 111
                        \]\vspace{0mm}
                        \[
                            \divTimeMapVector = (130,130,130)
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<7>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeVector = \{130,397\}
                        \]\vspace{0mm}
                        \[
                            model = 211
                        \]\vspace{0mm}
                        \[
                            \divTimeMapVector = (397,130,130)
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<8>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeVector = \{130,397\}
                        \]\vspace{0mm}
                        \[
                            model = 121
                        \]\vspace{0mm}
                        \[
                            \divTimeMapVector = (130,397,130)
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<9>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeVector = \{130,397\}
                        \]\vspace{0mm}
                        \[
                            model = 112
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{130,397\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<10-12>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeVector = \{130,397,260\}
                        \]\vspace{0mm}
                        \[
                            model = 312
                        \]\vspace{0mm}
                        \[
                            \divTimeMapVector = (260,130,397)
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        % \begin{onlyenv}<11>
        %     \begin{minipage}[c][0.5\textheight][c]{\linewidth}
        %         \begin{displaybox}[0.95\linewidth]
        %             \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
        %                 \[
        %                     \divTimeVector = \{\divTime{1},\ldots,\divTime{\divTimeNum}\}
        %                 \]\vspace{0mm}
        %                 \[
        %                     model = \divModel{i}
        %                 \]\vspace{0mm}
        %                 \[
        %                     \divTimeMapVector = (\divTimeMap{1},\ldots,\divTimeMap{\npairs{}})
        %                 \]\vspace{0mm}
        %             \end{minipage}
        %         \end{displaybox}
        %     \end{minipage}
        % \end{onlyenv}
        \begin{onlyenv}<13>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{mydescription}
                    \item[\alignmentVector] Sequence alignments
                    \item[\divTimeMapVector] Divergence times
                    \item[\divModel{}] Divergence model
                    \item[\allParameters{}] Gene trees
                    \item[\ ] Substitution parameters
                    \item[\ ] Demographic parameters
                    % \item[\allParameters] $(\divTimeMapVector, \geneTreeVector,
                    %     \hkyModelVector, \demographicParamVector)$
                \end{mydescription}
            \end{minipage}
        \end{onlyenv}
        \begin{uncoverenv}<11->
            \begin{minipage}[c][0.18\textheight][c]{\linewidth}
            \begin{flushleft}
                \small We want to infer \textcolor{blue}{\divModel{}} and
                \textcolor{blue}{\divTimeMapVector} given DNA sequence
                alignments
                \textcolor{blue}{\alignmentVector}
            \end{flushleft}
            \end{minipage}
        \end{uncoverenv}
        \column{.6\textwidth}
        \begin{minipage}[t][0.8\textheight][c]{\linewidth}
        \includegraphics<1>[height=6cm]{../images/dmc-cartoon-islands.pdf}
        \includegraphics<2>[height=6cm]{../images/dmc-cartoon-taxa.pdf}
        \includegraphics<3>[height=6cm]{../images/dmc-cartoon-sea-level.pdf}
        \includegraphics<4>[height=6cm]{../images/dmc-cartoon-shared-no-labels.pdf}
        \includegraphics<5-6>[height=6cm]{../images/dmc-cartoon-shared.pdf}
        \includegraphics<7>[height=6cm]{../images/dmc-cartoon-2-1.pdf}
        \includegraphics<8>[height=6cm]{../images/dmc-cartoon-2-2.pdf}
        \includegraphics<9>[height=6cm]{../images/dmc-cartoon-2-3.pdf}
        \includegraphics<10-11>[height=6cm]{../images/dmc-cartoon-general.pdf}
        \includegraphics<12->[height=6cm]{../images/dmc-cartoon-pop-general.pdf}
        \end{minipage}
    \end{columns}
\end{frame}

%   c. Testing for predicted patterns via model choice.
%       i. Bayesian model choice

\begin{frame}[t]
    \frametitle{Bayesian model choice}
    \begin{block}{\it Full model:}
        \begin{minipage}[c][3.8cm][c]{\linewidth}
            \begin{uncoverenv}<1->
            \[
                p(
                  \divTimeMapVector,
                  \allParameters{}
                  \given \alignmentVector, \divModel{i})
                  =
                \frac{
                    p(\alignmentVector \given
                      \divTimeMapVector,
                      \allParameters{},
                      \divModel{i}
                      )
                      p(
                        \divTimeMapVector,
                        \allParameters{}
                        \given \divModel{i}
                        )
                    }{p(\alignmentVector \given \divModel{i})}
            \]\vspace{-1mm}
            \end{uncoverenv}
            \begin{uncoverenv}<2->
            \[
                p(\alignmentVector \given \divModel{i}) =
                \int_{\divTimeMapVector} \int_{\allParameters{}}
                p(\alignmentVector \given \divTimeMapVector, \allParameters{}, \divModel{i})
                p(\divTimeMapVector, \allParameters{} \given \divModel{i})
                d\divTimeMapVector d\allParameters{}
            \]\vspace{-1mm}
            \end{uncoverenv}
            \begin{uncoverenv}<3->
            \[
                p(\divModel{i} \given \alignmentVector) =
                \frac{
                    p(\alignmentVector \given \divModel{i})
                    p(\divModel{i})
                }{
                    \sum_{i} p(\alignmentVector \given \divModel{i})
                    p(\divModel{i})
                }
            \]\vspace{-1mm}
            \end{uncoverenv}
        \end{minipage}
    \end{block}
    \vspace{1mm}
    \begin{uncoverenv}<4>
        \begin{center}
            \msb: Approximate Bayesian computation (ABC)

            % \vspace{-5mm}
            % \[ \alignmentVector \, \to \, \ssVectorObs \, \to \, \ssSpace\]
        \end{center}
    \end{uncoverenv}
    \barefootnote{\tiny\shortfullcite{Huang2011}. \hspace{2mm}\shortfullcite{Oaks2012}.}
\end{frame}

%   d. Method exists---applied it to comparative dataset from Philippines
%       i. Surprising results!
%       ii. Rather than publish, did some detective work

{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{../images/maps/se-asia-120.png}}
\begin{frame}
    \frametitle{Climate-driven diversification}    
    \begin{columns}
        \column{0.6\textwidth}
        \ \\

        \column{0.4\textwidth}

        \vspace{-2cm}

        \colorbox{white}{
            \begin{minipage}[t]{1.0\textwidth}
                \raggedright
                \textbf{Did repeated fragmentation of islands during
                    inter-glacial rises in sea level promote diversification?}
            \end{minipage}
        }
    \end{columns}
\end{frame}
}

\begin{frame}
    \frametitle{Climate-driven diversification}
\begin{columns}[c]
    \column{.5\textwidth}
        \vspace{-1cm}
        \begin{onlyenv}<1>
        \begin{minipage}[t][1.0\textheight][c]{\linewidth}
        \centerline{
        \includegraphics<1>[height=3cm]{/home/jamie/Dropbox/field-photos/people/rafe.jpg}
        \hspace{0.3mm}
        \includegraphics<1>[height=3cm]{/home/jamie/Dropbox/field-photos/people/cam.jpg}}
        \centerline{
        \includegraphics<1>[height=3cm]{/home/jamie/Dropbox/field-photos/people/charles.jpg}
        \hspace{0.3mm}
        \includegraphics<1>[height=3cm]{/home/jamie/Dropbox/field-photos/people/jake.jpg}}
        \end{minipage}
        \end{onlyenv}

        \begin{onlyenv}<2->
        \begin{minipage}[t][1.0\textheight][c]{\linewidth}
        \centerline{
        \includegraphics<2->[height=1.3cm]{../images/photos/crocidura-negrina-JAEsselstyn.jpg}
        \hspace{0.3mm}
        \includegraphics<2->[height=1.3cm]{../images/photos/crocidura-beatus-DSBalete.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<2->[height=1.3cm]{../images/photos/hipposideros-obscurus-MRMDuya.jpg}
        \hspace{0.3mm}
        \includegraphics<2->[height=1.3cm]{../images/photos/haplonycteris-fischeri-JHolden.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<2->[height=1.3cm]{../images/photos/sphenomorphus-abdictus-rmb.jpg}
        \hspace{0.3mm}
        \includegraphics<2->[height=1.3cm]{../images/photos/sphenomorphus-arborens-rmb.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<2->[height=1.3cm]{../images/photos/gekko-mindorensis.jpg}
        \hspace{0.3mm}
        \includegraphics<2->[height=1.3cm]{../images/photos/dendrelaphis-pictus-cds.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<2->[height=1.3cm]{../images/photos/cyrt-agusanensis.jpg}
        \hspace{0.3mm}
        \includegraphics<2->[height=1.3cm]{../images/photos/cyrt-annulatus-cds.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<2->[height=1.3cm]{../images/photos/limnonectes-magnus-cds.jpg}
        \hspace{0.3mm}
        \includegraphics<2->[height=1.3cm]{../images/photos/limnonectes-leytensis-rmb.jpg}}
        \end{minipage}
        \end{onlyenv}

    \column{.5\textwidth}
        \vspace{-1cm}
        \begin{minipage}[t][1.0\textheight][c]{\linewidth}
        \includegraphics<1-2>[width=\textwidth]{../images/maps/Philippines.png}
        \includegraphics<3>[width=\textwidth]{../images/maps/Philippines-negros_panay.png}
        \begin{onlyenv}<4->
            \begin{itemize}
                \item Strong support for simultaneous divergence of all 22 taxon pairs
                
                \item Posterior probability $>$ 0.96 \\
            
                \item $\sim$100,000--250,000 years ago
            \end{itemize}
        \end{onlyenv}
        \end{minipage}
\end{columns}
\end{frame}


%   e. Evaluated method via simulation
%       i. biased in alarming direction

\section{Simulation-based assessment of method}

\begin{frame}
    \frametitle{Simulation-based power analyses}
    % What is ``simultaneous''?
    \begin{myitemize}
        \item<1-> What if divergences were random?
        \item<2-> Simulate datasets in which all 22 divergence times are random
        \uncover<3->{
            \smallskip
            \begin{myitemize}
                \item $\divTime{} \sim U(0, \,0.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,1.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,2.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,5.0 \,MGA)$
                \smallskip
            \end{myitemize}
        \item<3-> $MGA$ = Millions of Generations Ago
        }
        \item<4-> Simulate 1000 datasets for each \divTime{} distribution
        \item<4-> Analyze all 4000 datasets as we did the empirical data
    \end{myitemize}
\end{frame}

\begin{frame}
    \frametitle{Simulation-based power analyses: Results}
        \centerline{
        \includegraphics[width=1.13\textwidth]{../images/old_old_power_psi_mode.pdf}}

    \begin{uncoverenv}<2->
        \begin{itemize}[<+->]
            \item \msb will often infer clustered divergences when divergences
                are random over millions of generations.

            \item The empirical results are likely spurious
        \end{itemize}
    \end{uncoverenv}

    \begin{uncoverenv}<3->
        \begin{block}{Objective:}
            Use principles of probability to extend \msb framework for improved
            estimation of shared evolutionary history
        \end{block}
    \end{uncoverenv}
    \barefootnote{\tiny\shortfullcite{Oaks2012}. \hspace{2mm}\shortfullcite{Oaks2014reply}}
\end{frame}


%   f. Rather than abandon approach---look for theoretical reasons for behavior
%       i. Strongly weighted posterior (marginal likelihood)
%       ii. Combinatorics
%   g. Developed new method---dpp-msbayes
%       i. Used DPP---common in Bayeisn nonparametrics
%           1. Flexible and mathematically convenient way of assigning prior
%           probability to all possible divergence models (partitions) directly
%           (avoid combinatoric issue)
%       ii. More flexible priors on parameters of model
%       iii. Implemented method in much more efficient multi-processing
%       framework.
%   e. Results---improved!
%   f. Empirical results...
%       i. Reducing bias and increasing power provides more "honest" estimate
%       of uncertainty
%       ii. A lot of uncertainty (Good! and Bad!)
%       iii. What next? More data and more power!

\begin{frame}[label=improvements]
    \frametitle{An improved method}
    Potential improvements:
    \begin{enumerate}
        \item Alternative priors on parameters that increase marginal
            likelihoods of rich models
        \item Alternative approach to modeling the temporal distribution of
            divergences
    \end{enumerate}
    \barefootnote{\tiny\shortfullcite{Oaks2012}. \hspace{2mm}\shortfullcite{Oaks2014reply}}
\end{frame}

\begin{frame}[t]
    \vspace{-2mm}
    \begin{displaybox}[5.5cm]
        \small
        \[
            p(X) = \int_{\theta} p(X \given \theta) p(\theta) \mathrm{d}\theta
        \]%\vspace{0mm}
    \end{displaybox}

    \vspace{-1mm}
    \begin{center}
        % \includegraphics<1>[height=7.8cm]{../images/marginal-plot-2d-uniform-prior.pdf}
        \includegraphics<2>[height=7.8cm]{../images/marginal-plot-3d-bare.png}
        \includegraphics<3>[height=7.8cm]{../images/marginal-plot-3d-constrained.png}
        \includegraphics<4>[height=7.8cm]{../images/marginal-plot-3d.png}
        \includegraphics<5>[height=7.8cm]{../images/marginal-plot-2d-uniform-prior.pdf}
        \includegraphics<6>[height=7.8cm]{../images/marginal-plot-2d.pdf}
    \end{center}
\end{frame}

\againframe{improvements}

\begin{frame}[t]
    \frametitle{Prior on divergence models}
    \begin{itemize}
        \item \msb uses a discrete uniform prior on the \emph{number} of
            divergence events
    \end{itemize}

    \begin{uncoverenv}<2->
    \begin{columns}
        \column{0.5\textwidth}
            \centerline{
                \includegraphics<2->[width=1.0\textwidth]{../images/number-of-div-models-22-unordered.pdf}}
        \column{0.5\textwidth}
            \centerline{
                \includegraphics<3->[width=1.0\textwidth]{../images/prob-of-div-models-22-unordered.pdf}}
    \end{columns}
    \end{uncoverenv}

    \begin{uncoverenv}<4->
    \begin{block}{\it Potential solution:}
        Place flexible prior directly on the sample space of divergence models
    \end{block}
    \end{uncoverenv}
\end{frame}


\begin{frame}
    \frametitle{New method: \dppmsbayes}
    \begin{itemize}[<+->]
        % \item<1-> Reparameterized the model implemented in \msb
        \item<1-> Replaced uniform priors on continuous parameters with gamma and
            beta distributions
        \item<2-> Dirichlet process prior (DPP) over all possible divergence
            models
        \item<3-> Develop multi-processing interface to improve performance for
            accommodating genomic datasets
        % \item<4-> Uniform prior over divergence models
    \end{itemize}
\end{frame}

\begin{frame}
    \frametitle{\dppmsbayes: Simulation-based assessment}
        \uncover<1->{
        Validation:\\

        \smallskip
        \begin{itemize}
            \item Simulate 50,000 datasets and analyze each under the same
                model
        \end{itemize}
        }

        \uncover<2->{
        \bigskip
        Robustness:\\
        \begin{itemize}
            \item Simulate datasets that violate model assumptions and analyze
                each of them
        \end{itemize}
        }

        \uncover<3->{
        \bigskip
        Done for \msb and \dppmsbayes
        }
\end{frame}

\begin{frame}
    \frametitle{\dppmsbayes: Validation results}
    \begin{columns}
        \column{0.5\textwidth}
            \includegraphics<1->[width=1.0\textwidth]{../images/validation-model-choice-old.pdf}
        \column{0.5\textwidth}
            \includegraphics<2->[width=1.0\textwidth]{../images/validation-model-choice-dpp.pdf}
    \end{columns}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}
    \frametitle{\dppmsbayes: Robustness results}
    \begin{columns}
        \column{0.5\textwidth}
            \centering{
            \includegraphics<1->[width=0.9\textwidth]{../images/validation-model-choice-old-violations.pdf}
            }
        \column{0.5\textwidth}
            \centering{
            \includegraphics<1->[width=0.9\textwidth]{../images/validation-model-choice-dpp-violations.pdf}
            }
    \end{columns}

    \begin{uncoverenv}<2->
    \begin{itemize}
        \item Improved model-choice accuracy and robustness
    \end{itemize}
    \end{uncoverenv}
    \vspace{-0.5cm}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}
    \frametitle{\dppmsbayes: Simulation-based power analyses}
    Does the new method have power to detect variation in divergence times?\\

    \bigskip
    \begin{myitemize}
        \item<2-> Simulate datasets in which all 22 divergence times are random
            \smallskip
            \begin{myitemize}
                \item $\divTime{} \sim U(0, \,0.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,1.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,2.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,5.0 \,MGA)$
                \smallskip
            \end{myitemize}
        \item<2-> $MGA$ = Millions of Generations Ago

        \item<3-> Simulate 1000 datasets for each \divTime{} distribution
        \item<3-> Analyze all 4000 datasets with \msb and \dppmsbayes
    \end{myitemize}
\end{frame}

\begin{frame}[t]
    \frametitle{\dppmsbayes: Power results}
        \begin{onlyenv}<1-2>
        \centerline{
        \includegraphics[width=1.13\textwidth]<1-2>{../images/power-psi-old-annotate-no-pp.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=1.13\textwidth]<2>{../images/power-psi-dpp-annotate-no-pp.pdf}}
        \end{onlyenv}

        \begin{onlyenv}<3->
        \centerline{
        \includegraphics[width=1.13\textwidth]<3->{../images/power-psi-old-annotate-pp.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=1.13\textwidth]<3->{../images/power-psi-dpp-annotate-pp.pdf}}
        \end{onlyenv}

    \begin{uncoverenv}<4->
    \begin{itemize}
        \item Improved power to detect temporal variation across divergences
    \end{itemize}
    \end{uncoverenv}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{../images/maps/se-asia-120.png}}
\begin{frame}
    \frametitle{Climate-driven diversification}    
    \begin{columns}
        \column{0.6\textwidth}
        \ \\

        \column{0.4\textwidth}

        \vspace{-2cm}

        \colorbox{white}{
            \begin{minipage}[t]{1.0\textwidth}
                \raggedright
                \textbf{Did repeated fragmentation of islands during
                    inter-glacial rises in sea level promote diversification?}
            \end{minipage}
        }
    \end{columns}
\end{frame}
}

\begin{frame}
    \frametitle{Climate-driven diversification}
\begin{columns}[c]
    \column{.5\textwidth}
        \vspace{-1cm}
        \begin{minipage}[t][1.0\textheight][c]{\linewidth}
        \centerline{
        \includegraphics<1->[height=1.3cm]{../images/photos/crocidura-negrina-JAEsselstyn.jpg}
        \hspace{0.3mm}
        \includegraphics<1->[height=1.3cm]{../images/photos/crocidura-beatus-DSBalete.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<1->[height=1.3cm]{../images/photos/hipposideros-obscurus-MRMDuya.jpg}
        \hspace{0.3mm}
        \includegraphics<1->[height=1.3cm]{../images/photos/haplonycteris-fischeri-JHolden.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<1->[height=1.3cm]{../images/photos/sphenomorphus-abdictus-rmb.jpg}
        \hspace{0.3mm}
        \includegraphics<1->[height=1.3cm]{../images/photos/sphenomorphus-arborens-rmb.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<1->[height=1.3cm]{../images/photos/gekko-mindorensis.jpg}
        \hspace{0.3mm}
        \includegraphics<1->[height=1.3cm]{../images/photos/dendrelaphis-pictus-cds.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<1->[height=1.3cm]{../images/photos/cyrt-agusanensis.jpg}
        \hspace{0.3mm}
        \includegraphics<1->[height=1.3cm]{../images/photos/cyrt-annulatus-cds.jpg}}
        % \vspace{0.5mm}
        \centerline{
        \includegraphics<1->[height=1.3cm]{../images/photos/limnonectes-magnus-cds.jpg}
        \hspace{0.3mm}
        \includegraphics<1->[height=1.3cm]{../images/photos/limnonectes-leytensis-rmb.jpg}}
        \end{minipage}

    \column{.5\textwidth}
        \vspace{-1cm}
        \begin{minipage}[t][1.0\textheight][c]{\linewidth}
        \includegraphics<1>[width=\textwidth]{../images/maps/Philippines.png}
        \end{minipage}
\end{columns}
\end{frame}

\begin{frame}
    \frametitle{Empirical results}
    \centerline{
    \includegraphics[width=\textwidth]{../../empirical-analyses/plots/philippines-dpp-psi-posterior-old-vs-dpp.pdf}}

    \begin{uncoverenv}<2->
    \begin{itemize}
        \item More ``honest'' estimate of posterior uncertainty
    \end{itemize}
    \end{uncoverenv}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}
    \frametitle{Empirical results}
    \centerline{
    \includegraphics[width=\textwidth]{../../empirical-analyses/plots/philippines-dpp-psi-posterior-prior.pdf}}
    \smallskip
    \centerline{
    \includegraphics[height=2cm]{../images/sea-level-only.pdf}}
\end{frame}

\begin{frame}
    \frametitle{Summary}
    \begin{itemize}
        \item<1-> New method for estimating shared evolutionary history shows
            improved:
            \begin{enumerate}
                \item<2-> Estimation of posterior uncertainty
                \item<2-> Model-choice accuracy
                \item<2-> Power to detect temporal variation across divergences
                \item<2-> Robustness to model violations
                \item<2-> Speed!
            \end{enumerate}

        \smallskip
        \item<3-> Climate-driven diversification model?
            \begin{itemize}
                \item<4-> Results consistent with prediction of clustered
                    divergences
                \item<4-> Results suggest multiple co-divergences
                \item<4-> However, there is a lot of uncertainty!
            \end{itemize}

        \smallskip
        \item<5-> What to do?
            \begin{itemize}
                \item<6-> More data!
                \item<6-> More power! 
            \end{itemize}
    \end{itemize}
\end{frame}

%   g. Collected thousands of loci from across the genome of almost 300
%   individuals of two genera of geckos from across the Philippine Islands
%       i. Prelim results with dpp-msbayes (can accommodate NGS data!)

\begin{frame}
    \frametitle{More data}
    \begin{columns}[c]
        \column{.5\textwidth}
        \begin{myitemize}
            \item Collecting genomic data from two gekkonid radiations
                co-distributed Southeast Asia
            \begin{myitemize}
                \item \emph{Cyrtodactylus}---12+ species
                \item \emph{Gekko}---13+ species
            \end{myitemize}
            \item Whole genome assemblies for each genus
            \item RAD-seq libraries constructed for $\approx150$ individuals
                from each genus
        \end{myitemize}
        \centerline{
        \includegraphics<1->[height=2.5cm]{/home/jamie/Dropbox/field-photos/misc/lee-and-me.jpg}
        \hspace{2mm}
        \includegraphics<1->[height=2.5cm]{/home/jamie/Dropbox/field-photos/people/rafe.jpg}}
        \column{.5\textwidth}
        \includegraphics[width=\textwidth]{../images/photos/cyrt-agusanensis.jpg} \quad
        \includegraphics[width=\textwidth]{../images/photos/gekko-mindorensis.jpg}
    \end{columns}
\end{frame}

\begin{frame}
    \frametitle{More data---preliminary results}
    Results for 1000 loci from 5 pairs of \emph{Gekko mindorensis} populations\\

    \begin{center}
        \includegraphics<1->[height=5.5cm]{../images/number-of-divs-gekko-mindorensis.pdf}
    \end{center}
\end{frame}


%   h. Currently working on full-likelihood implementation of the model.
%       i. Working with Jeet---has full-likelihood up and running---inefficient
%       ii. Take advantage of recent developments to analytically integrate
%       gene trees---might even be faster than simulations?!
%       iii. Uses ALL information in genetic data

\begin{frame}
    \frametitle{More power}
    \centerline{
    \includegraphics<1->[height=2cm]{/home/jamie/Dropbox/field-photos/people/mtholder.jpg}
    \hspace{0.6mm}
    \includegraphics<1->[height=2cm]{/home/jamie/Dropbox/field-photos/people/jeet2.jpg}
    \hspace{0.6mm}
    \includegraphics<1->[height=2cm]{/home/jamie/Dropbox/field-photos/people/vladimir.jpg}
    \hspace{0.6mm}
    \includegraphics<1->[height=2cm]{../images/revbayes.png}}

    \begin{itemize}[<+->]
        \item Full-likelihood Bayesian implementation \footnote{\tiny\shortfullcite{JeetDiss}}
            \begin{itemize}
                \item Uses all the information in the sequence data
                \item Applicable to much deeper timescales
                \item \ldots Currently inefficient
            \end{itemize}
        \item Working to incorporate new developments on analytical integration
            of gene trees \footnote{\tiny\shortfullcite{Bryant2012}}
            \begin{itemize}
                \item Very efficient numerical approximation of posterior 
                \item Applicable to NGS datasets
            \end{itemize}
    \end{itemize}
\end{frame}

\section{Future directions}

% III. Future work
%   a. Theory/computation
%       i. Extend full-likelihood approach to phylogenetic framework
%           1. Can infer relative mutation rates/pop sizes from data (fewer
%           assumptions)
%           2. Much better representation of diversification than pairwise
%           approach.
%       ii. Parameter estimation approach (Lindley's paradox).

\begin{frame}
    \frametitle{Future directions---Computational}
    \begin{itemize}
        \item<1-> Full-phylogenetic framework
    \end{itemize}
    
    \begin{center}
        \includegraphics<1>[height=6.5cm]{../images/dmc-cartoon-no-islands-pop-shared-middle-wide.pdf}
        \includegraphics<2>[height=6.5cm]{../images/dmc-cartoon-no-islands-pop-phylogeny.pdf}
    \end{center}
\end{frame}

%   b. Empirical
%       i. Gobi (NGS --> NSF)
%       ii. West-Central Africa
%           1. unfunded collaborator
%       iii. Compare diversification of mainland SE Asia to Oceanic Islands
%           1. Planning pre-proposal
%       iv. Long-term study of small island of sunda shelf
%           1. Tiny islands with pops of Eutropis multifasciata and
%           Lepidodactylus lugubris
%           2. Inundated during last interglacial ~120,000 years ago, spent the
%           better part of next 100k years as part of exposed sunda shelf, then
%           fragmented during current interglacial
%           3. Both species vary in reproductive mode (oviparity/ovoviviparity,
%           parthenogenesis), skinks in color patterns
%           4. Affects of recent, severe fragmentation on diversification and
%           demography
%           5. Comparative genomics to understand evolution of color and
%           reproductive mode
%           6. Evolutionary ecology---what ecological context influence
%           evolution asexuality/live-bearing/color.
{
\setbeamercolor{background canvas}{bg=black}
\setbeamercolor{whitetext}{fg=white}
\begin{frame}
    \frametitle{Processes of diversification}
    \begin{columns}[c]
        \column{.5\textwidth}
        {\usebeamercolor[fg]{whitetext}
        \begin{itemize}
            \item<1-> These comparative tools will allow evolutionary
                biologists to leverage genomic data to infer shared
                evolutionary history
            \item<2-> This will better our understanding of how community scale
                processes affect the generation and assembly of biodiversity
            \item<3-> Many exciting empirical systems for applying new methods!
        \end{itemize}
        }
        \column{.5\textwidth}
        \begin{figure}
            \begin{center}
            \includegraphics[width=\textwidth]{../images/earth-image.jpg}
            \end{center}
        \end{figure}
    \end{columns}
\end{frame}
}

\begin{frame}
    \frametitle{Future directions: Gobi Desert}
    \vspace{-2mm}
    \begin{columns}
        % \column{\dimexpr\paperwidth-10pt}
        \column{\dimexpr\paperwidth}
        \includegraphics[width=\textwidth]{/home/jamie/Dropbox/field-photos/gobi/mongolia-auto-shop.jpg}
    \end{columns}
\end{frame}

\begin{frame}
    \frametitle{Future directions: Gobi Desert}
    \vspace{-2mm}
    \begin{columns}
        % \column{\dimexpr\paperwidth-10pt}
        \column{\dimexpr\paperwidth}
        \includegraphics[width=\textwidth]{/home/jamie/Dropbox/field-photos/gobi/mongolia-teratoscincus-przewalskii-landscape.jpg}
    \end{columns}
\end{frame}

\begin{frame}[t]
    \frametitle{Future directions: West-Central Africa}
    \begin{center}
        \includegraphics<1->[height={3cm}]{/home/jamie/Dropbox/field-photos/africa/Figure2_WAmapForestDist.png}
        \hspace{0.3mm}
        \includegraphics<1->[height={3cm}]{/home/jamie/Dropbox/field-photos/africa/Figure4_herps.jpeg}
    \end{center}

    \only<1>{
    \begin{center}
        \includegraphics[height={3.5cm}]{/home/jamie/Dropbox/field-photos/people/adam.png}
        \hspace{0.3mm}
        \includegraphics[height={3.5cm}]{/home/jamie/Dropbox/field-photos/people/fujita.jpg}
    \end{center}
    }

    \uncover<2->{
    \begin{center}
        \includegraphics[height=4.5cm]{../images/number-of-divs-africa.pdf}
    \end{center}
    }
\end{frame}


\begin{frame}
    \frametitle{``Micro''-islands of the Sunda Shelf}
    \vspace{-2mm}
    \begin{columns}
        % \column{\dimexpr\paperwidth-10pt}
        \column{\dimexpr\paperwidth}
        \includegraphics[width=\textwidth]{/home/jamie/Dropbox/field-photos/malaysia/malaysia-burungs.jpg}
    \end{columns}
\end{frame}

{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{../images/maps/se-asia-present.png}}
\begin{frame}
    \frametitle{``Micro''-islands of the Sunda Shelf}
\end{frame}
}

{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{../images/maps/se-asia-120.png}}
\begin{frame}
    \frametitle{``Micro''-islands of the Sunda Shelf}
\end{frame}
}

\begin{frame}
    \frametitle{Everything is on GitHub\ldots}
    Software:\\
    \begin{itemize}
        \item \texttt{dpp-msbayes}:
            \href{https://github.com/joaks1/dpp-msbayes}{\tt
            https://github.com/joaks1/dpp-msbayes}

        \item \texttt{PyMsBayes}:
            \href{https://github.com/joaks1/PyMsBayes}{\tt
            https://github.com/joaks1/PyMsBayes}

        \item ABACUS: Approximate BAyesian C UtilitieS.
            \href{https://github.com/joaks1/abacus}{\tt
            https://github.com/joaks1/abacus}
    \end{itemize}

    \medskip
    Open-Science Notebook:\\
    \begin{itemize}
        \item \texttt{msbayes-experiments}:
            \href{https://github.com/joaks1/msbayes-experiments}{\tt
            https://github.com/joaks1/msbayes-experiments}
    \end{itemize}
\end{frame}


\begin{frame}
    \frametitle{Acknowledgments}
    \begin{columns}[t]
        \column{.5\textwidth}
            {\bf Ideas and feedback:}
            \begin{myitemize}
                \item Leach\'{e} Lab
                \item Minin Lab
                \item Holder Lab
                \item Brown Lab/KU Herpetology
                \item Melissa Callahan
            \end{myitemize}
            \smallskip
            {\bf Computation:}\\
            \includegraphics<1->[height={8mm}]{../images/iplant.jpg}
            \hspace{0.5mm}
            \includegraphics<1->[height={8mm}]{../images/kuittc.png}
            \hspace{0.5mm}
            \includegraphics<1->[height={8mm}]{../images/uw.png}\\

        \column{.5\textwidth}
            {\bf Funding:}\\
            \includegraphics<1->[height={8mm}]{../images/nsf.jpg}
            \hspace{0.5mm}
            \includegraphics<1->[height={8mm}]{../images/ngs.jpg}
            \hspace{0.5mm}
            \includegraphics<1->[height={8mm}]{../images/ssb.png}\\

            \smallskip
            {\bf Photo credits:}
            \begin{myitemize}
                \item Rafe Brown, Cam Siler, Jesse Grismer, \& Jake Esselstyn
                \item FMNH Philippine Mammal Website:
                    \begin{myitemize}
                        \item D.S.\ Balete, M.R.M.\ Duya, \& J.\ Holden
                    \end{myitemize}
                \item PhyloPic!
            \end{myitemize}
    \end{columns}
\end{frame}

\begin{frame}
    \frametitle{Questions?}    
    \begin{columns}[c]
        \column{.5\textwidth}
        \begin{center}
            {
            \Large
            \href{mailto:joaks1@gmail.com}{\tt joaks1@gmail.com}
            }
        \end{center}
        \column{.5\textwidth}
            \begin{figure}
                \begin{center}
                \includegraphics[width=\textwidth]{../images/darwin-tol-copyright-boris-kulikov-2007.jpg}
                \caption{\tiny \copyright~2007 Boris Kulikov \href{http://boris-kulikov.blogspot.com/}{boris-kulikov.blogspot.com}}
                \end{center}
            \end{figure}
    \end{columns}
\end{frame}

% Extra slides

\begin{frame}[noframenumbering]
    \frametitle{Easy as ABC}
    \centerline{
    \includegraphics<1>[width=\textwidth]{../images/rejection-sampling-observed.pdf}
    \includegraphics<2>[width=\textwidth]{../images/rejection-sampling-tolerance.pdf}
    \includegraphics<3>[width=\textwidth]{../images/rejection-sampling-10.pdf}
    \includegraphics<4>[width=\textwidth]{../images/rejection-sampling-100.pdf}
    \includegraphics<5>[width=\textwidth]{../images/rejection-sampling-200.pdf}
    \includegraphics<6>[width=\textwidth]{../images/rejection-sampling-500.pdf}
    \includegraphics<7>[width=\textwidth]{../images/rejection-sampling-1000.pdf}
    \includegraphics<8>[width=\textwidth]{../images/rejection-sampling-10000.pdf}
    \includegraphics<9>[width=\textwidth]{../images/rejection-sampling-20000.pdf}
    \includegraphics<10>[width=\textwidth]{../images/rejection-sampling-20000-post.pdf}
    }
\end{frame}

\begin{frame}[noframenumbering]
    \frametitle{Causes of bias: Insufficient sampling}
    \begin{itemize}
        \item Models with more parameter space are less densely sampled
        \item Could explain bias toward small models in extreme cases
        \item {\bf Predicts large variance in posterior estimates}
        \begin{itemize}
            \item We explored empirical and simulation-based analyses with
                2, 5, and 10 million prior samples, and estimates were
                very similar
        \end{itemize}
    \end{itemize}
    \smallskip
    \centerline{
    \includegraphics[width=\textwidth]{../images/omega_over_sampling.pdf}}
\end{frame}

\begin{frame}[noframenumbering]
    \frametitle{\dppmsbayes: Simulation results}
    % \vspace{1cm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_old_power_psi_mode.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_u-shaped_power_psi_mode_headless.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_dpp_power_psi_mode_headless.pdf}}
    % \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}[noframenumbering]
    \frametitle{\dppmsbayes: Simulation results}
    % \vspace{1cm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_old_power_omega_median.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_u-shaped_power_omega_median_headless.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_dpp_power_omega_median_headless.pdf}}
    % \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}[noframenumbering]
    \frametitle{Empirical results: Philippine diversification}
    \centerline{
    \includegraphics[width=0.8\textwidth]{../../empirical-analyses/plots/philippines-dpp-psi-posterior-old-vs-dpp-with-prior.pdf}}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}


\end{document}

