\input{../utils/preamble.tex}
\input{../../manuscripts/utils/macros.tex}
\newcommand{\allParameters}[1]{\ensuremath{\theta_{#1}}\xspace}
\bibliography{../../manuscripts/bib/references}

\title[Estimating shared history]{An Improved Approximate-Bayesian Method for Estimating Shared
Evolutionary History}

\author[J.\ Oaks]{
    Jamie R.\ Oaks\inst{1,2}
}
\institute[University of Washington]{
    \inst{1}%
        Department of Ecology and Evolutionary Biology, University of Kansas
    \and
    \inst{2}%
        Department of Biology, University of Washington
}

% \date{\today}
\date{June 21, 2014}

\begin{document}
\maketitle

% \begin{frame}
% \frametitle{Outline}
% \tableofcontents
% \end{frame}

% \section{Intro}
% \begin{frame}
% \frametitle{Table of Contents}
% \tableofcontents[currentsection]
% \end{frame}

{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{../images/maps/se-asia-120.png}}
\begin{frame}
    \frametitle{Southeast Asia}    
\end{frame}
}

\begin{frame}
    \frametitle{Processes of diversification}
        \begin{itemize}
            \item<1-> Large-scale geological and climatic processes are
                important in biodiversification and community assembly
            \item<2-> Accounting for such processes will better our
                understanding of biodiversity
            \item<3-> We need methods for inferring evolutionary patterns
                predicted by historical events from contemporary population
        \end{itemize}
\end{frame}

%TODO: NEED GENERAL INTRO

\begin{frame}
    \frametitle<1-6>{Community scale processes}
    \frametitle<7->{Divergence model choice}
    \begin{columns}[c]
        \column{.4\textwidth}
        \begin{onlyenv}<1-6>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<7>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeMapVector = (\divTimeMap{1},\divTimeMap{2},\divTimeMap{3})
                        \]\vspace{0mm}
                        \[
                            model = 111
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{\divTime{1}\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<8>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeMapVector = (260,260,260)
                        \]\vspace{0mm}
                        \[
                            model = 111
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{260\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<9>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeMapVector = (397,260,260)
                        \]\vspace{0mm}
                        \[
                            model = 211
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{260,397\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<10>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeMapVector = (260,397,260)
                        \]\vspace{0mm}
                        \[
                            model = 121
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{260,397\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<11>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeMapVector = (260,260,397)
                        \]\vspace{0mm}
                        \[
                            model = 112
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{260,397\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<12>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeMapVector = (260,95,397)
                        \]\vspace{0mm}
                        \[
                            model = 123
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{260,95,397\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<13>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeMapVector = (260,260,260)
                        \]\vspace{0mm}
                        \[
                            model = 111
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{260\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<14-16>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{displaybox}[0.95\linewidth]
                    \begin{minipage}[c][0.45\textheight][c]{0.95\linewidth}
                        \[
                            \divTimeMapVector = (\divTimeMap{1},\ldots,\divTimeMap{\npairs{}})
                        \]\vspace{0mm}
                        \[
                            model = \divModel{i}
                        \]\vspace{0mm}
                        \[
                            \divTimeVector = \{\divTime{1},\ldots,\divTime{\divTimeNum}\}
                        \]\vspace{0mm}
                    \end{minipage}
                \end{displaybox}
            \end{minipage}
        \end{onlyenv}
        \begin{onlyenv}<17>
            \begin{minipage}[c][0.5\textheight][c]{\linewidth}
                \begin{mydescription}
                    \item[\alignmentVector] Sequence alignments
                    \item[\divTimeMapVector] Divergence times
                    \item[\divModel{}] Divergence model
                    \item[\geneTreeVector] Gene trees
                    \item[\hkyModelVector] Substitution parameters
                    \item[\demographicParamVector] Demographic parameters
                    % \item[\allParameters] $(\divTimeMapVector, \geneTreeVector,
                    %     \hkyModelVector, \demographicParamVector)$
                \end{mydescription}
            \end{minipage}
        \end{onlyenv}
        \begin{uncoverenv}<15->
            \begin{minipage}[c][0.18\textheight][c]{\linewidth}
            \begin{flushleft}
                \small We want to infer \textcolor{blue}{\divModel{}} and
                \textcolor{blue}{\divTimeMapVector} given DNA sequence
                alignments
                \textcolor{blue}{\alignmentVector}
            \end{flushleft}
            \end{minipage}
        \end{uncoverenv}
        \column{.6\textwidth}
        \begin{minipage}[t][0.8\textheight][c]{\linewidth}
        \includegraphics<1>[height=6cm]{../images/community-ellipse-small.pdf}
        \includegraphics<2>[height=6cm]{../images/community-ellipse-barrier-small.pdf}
        \includegraphics<3>[height=6cm]{../images/community-ellipse-split-small.pdf}
        \includegraphics<4>[height=6cm]{../images/div-model-cartoon-taxa.pdf}
        \includegraphics<5>[height=6cm]{../images/div-model-cartoon-shared-no-labels.pdf}
        \includegraphics<6-8>[height=6cm]{../images/div-model-cartoon-shared.pdf}
        \includegraphics<9>[height=6cm]{../images/div-model-cartoon-2-1.pdf}
        \includegraphics<10>[height=6cm]{../images/div-model-cartoon-2-2.pdf}
        \includegraphics<11>[height=6cm]{../images/div-model-cartoon-2-3.pdf}
        \includegraphics<12>[height=6cm]{../images/div-model-cartoon-general.pdf}
        \includegraphics<13-15>[height=6cm]{../images/div-model-cartoon-shared.pdf}
        \includegraphics<16->[height=6cm]{../images/div-model-cartoon-sp-tree-shared.pdf}
        \end{minipage}
    \end{columns}
\end{frame}

\begin{frame}[t]
    \frametitle{Bayesian model choice}
    \begin{block}{\it Full model:}
        \begin{minipage}[c][3.8cm][c]{\linewidth}
            \begin{uncoverenv}<1->
            \[
                p(
                  \divTimeMapVector,
                  \geneTreeVector,
                  \hkyModelVector,
                  \demographicParamVector
                  \given \alignmentVector, \divModel{i})
                  =
                \frac{
                    p(\alignmentVector \given
                      \divTimeMapVector,
                      \geneTreeVector,
                      \hkyModelVector,
                      \demographicParamVector,
                      \divModel{i}
                      )
                      p(
                        \divTimeMapVector,
                        \geneTreeVector,
                        \hkyModelVector,
                        \demographicParamVector
                        \given \divModel{i}
                        )
                    }{p(\alignmentVector \given \divModel{i})}
            \]\vspace{-1mm}
            \end{uncoverenv}
            \begin{uncoverenv}<2->
            \[
                p(\alignmentVector \given \divModel{i}) =
                \int_{\allParameters{i}}
                p(\alignmentVector \given \allParameters{i}, \divModel{i})
                p(\allParameters{i} \given \divModel{i})
                d\allParameters{i}
            \]\vspace{-1mm}
            \end{uncoverenv}
            \begin{uncoverenv}<3->
            \[
                p(\divModel{i} \given \alignmentVector) =
                \frac{
                    p(\alignmentVector \given \divModel{i})
                    p(\divModel{i})
                }{
                    \sum_{i} p(\alignmentVector \given \divModel{i})
                    p(\divModel{i})
                }
            \]\vspace{-1mm}
            \end{uncoverenv}
        \end{minipage}
    \end{block}
    % \end{minipage}
    % \begin{minipage}[c][2cm][c]{\linewidth}
    %     \begin{block}{\it Full Model:}
    %         \begin{minipage}[c][1.7cm][c]{\linewidth}
    %             \[
    %                 p(
    %                   \divTimeMapVector,
    %                   \geneTreeVector,
    %                   \hkyModelVector,
    %                   \demographicParamVector
    %                   \given \alignmentVector)
    %                   =
    %                 \frac{
    %                     p(\alignmentVector \given
    %                       \divTimeMapVector,
    %                       \geneTreeVector,
    %                       \hkyModelVector,
    %                       \demographicParamVector
    %                       )
    %                     p(
    %                       \divTimeMapVector,
    %                       \geneTreeVector,
    %                       \hkyModelVector,
    %                       \demographicParamVector
    %                       )
    %                     }{p(\alignmentVector)}
    %             \]\vspace{0mm}
    %         \end{minipage}
    %     \end{block}
    % \end{minipage}
    % \begin{minipage}[c][2cm][c]{\linewidth}
    %     \begin{block}{\it Full Model:}
    %         \begin{minipage}[c][1.7cm][c]{\linewidth}
    %             \[
    %                 p(
    %                   \divTimeMapVector,
    %                   \geneTreeVector,
    %                   \hkyModelVector,
    %                   \demographicParamVector
    %                   \given \alignmentVector)
    %                   =
    %                 \frac{p(\alignmentVector \given
    %                         \geneTreeVector,
    %                         \hkyModelVector)
    %                       p(\geneTreeVector \given
    %                         \divTimeMapVector,
    %                         \demographicParamVector)
    %                       p(\divTimeMapVector)
    %                       p(\hkyModelVector)
    %                       p(\demographicParamVector)
    %                     }{p(\alignmentVector)}
    %             \]\vspace{0mm}
    %         \end{minipage}
    %     \end{block}
    % \end{minipage}
    % \begin{onlyenv}<1->
    %     \bigskip
    %     \begin{mydescription}
    %             \item[\alignmentVector] Sequence alignments
    %             \item[\divTimeMapVector] Divergence times
    %             \item[\geneTreeVector] Gene trees
    %             \item[\hkyModelVector] Substitution parameters
    %             \item[\demographicParamVector] Demographic parameters
    %     \end{mydescription}
    % \end{onlyenv}
    \vspace{1mm}
    \begin{uncoverenv}<4>
        \begin{center}
            \msb: Approximate Bayesian computation (ABC)

            % \vspace{-5mm}
            % \[ \alignmentVector \, \to \, \ssVectorObs \, \to \, \ssSpace\]
        \end{center}
    \end{uncoverenv}
    \barefootnote{\tiny\shortfullcite{Huang2011}. \hspace{2mm}\shortfullcite{Oaks2012}.}
\end{frame}

\begin{frame}[t]
    \frametitle{The \msb model}
    \begin{minipage}[c][0.18\textheight][c]{\linewidth}
        % \begin{itemize}
        \msb will often infer clustered divergences when divergences are random
        over millions of generations.
        % \end{itemize}
    \end{minipage}

%TODO: NEED TRANSITION TO OBJECTIVES

    \begin{onlyenv}<1>
        \vspace{-3mm}
        \begin{center}
            \includegraphics[width=0.5\textwidth]{../images/old_old_power_psi_mode_last.pdf}
        \end{center}
    \end{onlyenv}
    \begin{onlyenv}<2->
        \begin{block}{Objective:}
            Use principles of probability to extend \msb framework for improved
            estimation of shared evolutionary history
        \end{block}
    \end{onlyenv}
    \barefootnote{\tiny\shortfullcite{Oaks2012}. \hspace{2mm}\shortfullcite{Oaks2014reply}}
\end{frame}

\begin{frame}
    \frametitle{An improved method}
    Potential improvements:
    \begin{enumerate}
        \item Alternative priors on parameters that increase marginal
            likelihoods of rich models
        \item Alternative prior over divergence models
        % \item Broad uniform priors on many of the model's parameters, including
        %     divergence times
        % \begin{itemize}
        %     \item Small marginal likelihoods of rich models
        % \end{itemize}
        % % \item Uniform priors on many of the model's parameters, including
        % %     divergence times
        % \item Prior on divergence models that, coupled with \#1, favors
        %     models with few divergence events
    \end{enumerate}
    \barefootnote{\tiny\shortfullcite{Oaks2012}. \hspace{2mm}\shortfullcite{Oaks2014reply}}
\end{frame}

\begin{frame}[t]
    \vspace{-2mm}
    \begin{displaybox}[5.5cm]
        \small
        \[
            p(X) = \int_{\theta} p(X \given \theta) p(\theta) \mathrm{d}\theta
        \]%\vspace{0mm}
    \end{displaybox}

    \vspace{-1mm}
    \begin{center}
        % \includegraphics<1>[height=7.8cm]{../images/marginal-plot-2d-uniform-prior.pdf}
        \includegraphics<2>[height=7.8cm]{../images/marginal-plot-3d.png}
        \includegraphics<3>[height=7.8cm]{../images/marginal-plot-2d-uniform-prior.pdf}
        \includegraphics<4>[height=7.8cm]{../images/marginal-plot-2d.pdf}
    \end{center}
\end{frame}

\begin{frame}
    \frametitle{An improved method}
    Potential improvements:
    \begin{enumerate}
        \item Alternative priors on model parameters that increase marginal
            likelihoods of rich models
        \item Alternative prior over divergence models
    \end{enumerate}
    \barefootnote{\tiny\shortfullcite{Oaks2012}. \hspace{2mm}\shortfullcite{Oaks2014reply}}
\end{frame}

\begin{frame}
    \frametitle{Prior on divergence models}
    \begin{itemize}
        \item \msb uses a discrete uniform prior on the \emph{number} of
            divergence events
    \end{itemize}
    % \smallskip
    \centerline{
        \includegraphics<1->[width=\textwidth]{../images/partition_numbers.pdf}}
    \begin{block}<2->{\it Potential solution:}
        Place flexible prior directly on the sample space of divergence models
    \end{block}
\end{frame}


\begin{frame}
    \frametitle{New method: \dppmsbayes}
    \begin{itemize}
        % \item<1-> Reparameterized the model implemented in \msb
        \item<1-> Replaced uniform priors on continuous parameters with gamma and
            beta distributions
        \item<1-> Dirichlet process prior (DPP) over all possible divergence
            models
        % \item<4-> Uniform prior over divergence models
    \end{itemize}
\end{frame}

\begin{frame}
    \frametitle{\dppmsbayes: Simulation-based assessment}
        Simulate 50,000 datasets under three models\\
        \smallskip
        \begin{description}
            \item[$M_{msBayes}$]
                \begin{itemize}
                    \item U-shaped prior on divergence models
                    \item Uniform priors on continuous parameters
                \end{itemize}
            \smallskip
            \item[$M_{Ushaped}$]
                \begin{itemize}
                    \item U-shaped prior on divergence models
                    \item Gamma priors on continuous parameters
                \end{itemize}
            \smallskip
            % \item[$M_{Uniform}$]
            %     \begin{itemize}
            %         \item Uniform prior on divergence models
            %         \item Gamma priors on continuous parameters
            %     \end{itemize}
            % \smallskip
            \item[$M_{DPP}$]
                \begin{itemize}
                    \item DPP prior on divergence models
                    \item Gamma priors on continuous parameters
                \end{itemize}
        \end{description}
        \smallskip
        Analyze all datasets under each of the models
\end{frame}

\begin{frame}
    \frametitle{\dppmsbayes: Simulation results}
    \centerline{
        \includegraphics<1>[width=0.85\textwidth]{../images/validation-model-choice-old-dpp.pdf}
        \includegraphics<2>[width=1.13\textwidth]{../images/validation-model-choice-old-dpp-full.pdf}}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}
    \frametitle{\dppmsbayes: Simulation-based power analyses}
    \begin{myitemize}
        \item Simulate datasets in which all 22 divergence times are random
        \uncover{
            \smallskip
            \begin{myitemize}
                \item $\divTime{} \sim U(0, \,0.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,1.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,2.5 \,MGA)$
                \smallskip
                \item $\divTime{} \sim U(0, \,5.0 \,MGA)$
                \smallskip
            \end{myitemize}
        \item $MGA$ = Millions of Generations Ago
        }
        \item Simulate 1000 datasets for each \divTime{} distribution
        % \item Analyze all 4000 datasets as we did the empirical data
        \item Analyze all 4000 datasets under models $M_{msBayes}$, $M_{Ushaped}$, and $M_{DPP}$
    \end{myitemize}
\end{frame}

\begin{frame}[t]
    \frametitle{\dppmsbayes: Power results}
    \vspace{1cm}
        \centerline{
        \includegraphics[width=1.13\textwidth]{../images/old_old_power_psi_mode.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=1.13\textwidth]{../images/old_dpp_power_psi_mode_headless.pdf}}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}[t]
    \frametitle{\dppmsbayes: Power results}
    \vspace{1cm}
        \centerline{
        \includegraphics[width=1.13\textwidth]{../images/old_old_power_psi_prob.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=1.13\textwidth]{../images/old_dpp_power_psi_prob_headless.pdf}}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}
    \frametitle{\dppmsbayes: Power results}
    % \vspace{1cm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_old_power_psi_prob.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_u-shaped_power_psi_prob_headless.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_dpp_power_psi_prob_headless.pdf}}
    % \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}
    \frametitle{Empirical application}
    \begin{columns}[c]
        \column{.5\textwidth}
        Did fragmentation of Philippine Islands during inter-glacial rises in
        sea level promote diversification?\\
        \begin{center}
            \includegraphics<1>[height=1.6cm]{../images/photos/gekko-mindorensis.jpg}
            \hspace{0.3mm}
            \includegraphics<1>[height=1.6cm]{../images/photos/sphenomorphus-arborens-rmb.jpg}

            \vspace{0.7mm}
            \includegraphics<1>[height=1.6cm]{../images/photos/dendrelaphis-pictus-cds.jpg}
            \hspace{0.3mm}
            \includegraphics<1>[height=1.6cm]{../images/photos/limnonectes-leytensis-rmb.jpg}

            \vspace{0.7mm}
            \includegraphics<1>[height=1.4cm]{../images/photos/hipposideros-obscurus-MRMDuya.jpg}
            \hspace{0.3mm}
            % \includegraphics<1>[height=1.2cm]{../images/photos/haplonycteris-fischeri-JHolden.jpg}
            % \hspace{0.1mm}
            \includegraphics<1>[height=1.4cm]{../images/photos/crocidura-negrina-JAEsselstyn.jpg}
        \end{center}
        \column{.5\textwidth}
        \includegraphics[width=\textwidth]{../images/maps/Philippines.png}
    \end{columns}
\end{frame}

\begin{frame}
    \frametitle{Empirical results: Philippine diversification}
    \centerline{
    \includegraphics[width=0.8\textwidth]{../../empirical-analyses/plots/philippines-dpp-psi-posterior-old-vs-dpp-with-prior.pdf}}
    \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\section{Conclusions}

\begin{frame}
    \frametitle{Conclusions}
    \begin{itemize}
        \item<1-> New method for estimating shared evolutionary history shows
            improved
        \begin{enumerate}
            \item Estimates of posterior uncertainty
            \item Model-choice accuracy
            \item Power to detect temporal variation across divergences
            \item Robustness to model violations
        \end{enumerate}

        \item<2-> Improvement is due to
        \begin{enumerate}
            \item Alternative priors on model parameters avoid prohibitively
                small marginal likelihoods for models with more divergence
                events
            \item Nonparametric prior on the temporal distribution of
                divergences avoids combinatorial ``interaction'' with
                divergence-time priors
        \end{enumerate}
    \end{itemize}
\end{frame}

\begin{frame}
    \frametitle{Caveats}
    \begin{itemize}
        \item<1-> Estimating a very rich (600+ parameters for 22 taxa) model
            using limited information from the data

        \begin{itemize}
            \item Likely sensitive to prior assumptions
            \item Be skeptical of strongly supported results
        \end{itemize}

        \item<2-> ABC can be inaccurate for model choice
            \footnote{\tiny\shortfullcite{Robert2011}}
    \end{itemize}
\end{frame}

\begin{frame}
    \frametitle{Recommendations}
    For Bayesian model choice, choose priors carefully\\
    \bigskip
    ABC model choice estimates should be accompanied by:
    \begin{enumerate}
        \item Simulation-based power analyses
        \item Assessment of prior sensitivity
    \end{enumerate}
\end{frame}

\begin{frame}[t]
    \frametitle{Future directions}
    \begin{itemize}
        \item<1-> Full-likelihood Bayesian approach \footnote{\tiny\shortfullcite{JeetDiss}}
        \item<2-> Full-phylogenetic framework
    \end{itemize}
    
    \begin{center}
        \includegraphics<2>[height=5.5cm]{../images/div-model-cartoon-sp-tree-shared-wide.pdf}
        \includegraphics<3>[height=5.5cm]{../images/div-model-cartoon-sp-phylogeny.pdf}
    \end{center}
\end{frame}

\begin{frame}
    \frametitle{Software}
    Everything is on GitHub\ldots\\
    \smallskip
    \begin{itemize}
        \item \texttt{dpp-msbayes}:
            \href{https://github.com/joaks1/dpp-msbayes}{\tt
            https://github.com/joaks1/dpp-msbayes}

        \item \texttt{PyMsBayes}:
            \href{https://github.com/joaks1/PyMsBayes}{\tt
            https://github.com/joaks1/PyMsBayes}

        \item ABACUS: Approximate BAyesian C UtilitieS.
            \href{https://github.com/joaks1/abacus}{\tt
            https://github.com/joaks1/abacus}
    \end{itemize}
\end{frame}

\begin{frame}
    \frametitle{Open Notebook Science}
    Everything is on GitHub\ldots\\
    \smallskip
    \begin{itemize}
        \item \texttt{msbayes-experiments}:
            \href{https://github.com/joaks1/msbayes-experiments}{\tt
            https://github.com/joaks1/msbayes-experiments}
        % \item joaks1@gmail.com
    \end{itemize}
\end{frame}

\begin{frame}
    \frametitle{Acknowledgments}
    \begin{columns}[t]
        \column{.5\textwidth}
            {\bf Ideas and feedback:}
            \begin{myitemize}
                \item Holder Lab
                \item KU Herpetology
                \item Melissa Callahan
            \end{myitemize}
            \smallskip
            {\bf Computation:}
            \begin{myitemize}
                \item KU ITTC
                \item KU Computing Center
                \item iPlant
            \end{myitemize}
        \column{.5\textwidth}
            {\bf Funding:}
            \begin{myitemize}
                \item NSF
                \item KU Grad Studies, EEB \& BI
                \item SSB
                \item Sigma Xi
            \end{myitemize}
            \smallskip
            {\bf Photo credits:}
            \begin{myitemize}
                \item Rafe Brown, Cam Siler, \& Jake Esselstyn
                \item FMNH Philippine Mammal Website:
                    \begin{myitemize}
                        \item D.S.\ Balete, M.R.M.\ Duya, \& J.\ Holden
                    \end{myitemize}
            \end{myitemize}
    \end{columns}
\end{frame}

{
\usebackgroundtemplate{\includegraphics[width=\paperwidth]{../images/maps/se-asia-120.png}}
\begin{frame}
    \frametitle{Questions?}    
\end{frame}
}

% Extra slides

\begin{frame}[noframenumbering]
    \frametitle{Causes of bias: Insufficient sampling}
    \begin{itemize}
        \item Models with more parameter space are less densely sampled
        \item Could explain bias toward small models in extreme cases
        \item {\bf Predicts large variance in posterior estimates}
        \begin{itemize}
            \item We explored empirical and simulation-based analyses with
                2, 5, and 10 million prior samples, and estimates were
                very similar
        \end{itemize}
    \end{itemize}
    \smallskip
    \centerline{
    \includegraphics[width=\textwidth]{../images/omega_over_sampling.pdf}}
\end{frame}

\begin{frame}[noframenumbering]
    \frametitle{\dppmsbayes: Simulation results}
    % \vspace{1cm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_old_power_psi_mode.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_u-shaped_power_psi_mode_headless.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_dpp_power_psi_mode_headless.pdf}}
    % \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}

\begin{frame}[noframenumbering]
    \frametitle{\dppmsbayes: Simulation results}
    % \vspace{1cm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_old_power_omega_median.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_u-shaped_power_omega_median_headless.pdf}}
        \vspace{0mm}
        \centerline{
        \includegraphics[width=\textwidth]{../images/old_dpp_power_omega_median_headless.pdf}}
    % \barefootnote{\shortfullcite{Oaks2014dpp}}
\end{frame}


\end{document}

